frontend/src/metabase/ui/components/inputs/TextInput/TextInput.styled.tsx

```diff
@@ -1,22 +1,10 @@
 import type { MantineThemeOverride } from "@mantine/core";
```

Comment:
These changes have a wide blast radius. I'm happy to pull them out into a separate PR if that makes it easier to review.

The idea is that we make `Input.Wrapper` available so we can use it to build our own form components. For that to work the custom wrapper styles need to be applied to it too, and not just to the `wrapper` when used in `Input`.